src/js/utils/styles.js

```diff
@@ -397,14 +397,35 @@ const placeholderColor = css`
 
 const placeholderStyle = css`
   &::-webkit-input-placeholder {
+    ${props =>
+      props.theme.global.input.placeholder &&
+      props.theme.global.input.placeholder.font.weight &&
```

Comment:
Thanks for the review here. I think for now we are actually going to just add `global.input.extend` to the theme options and handle it that way since this is a fairly specific use case. These styles will be handled for the DS use case via the extend.